server/sandbox_stop.go

```diff
@@ -35,6 +36,15 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 		return nil, err
 	}
 	if _, err := os.Stat(netnsPath); err == nil {
+		if err2 := s.hostportManager.Remove(sb.id, &hostport.PodPortMapping{
+			Name:         sb.name,
+			PortMappings: sb.portMappings,
+			HostNetwork:  false,
+		}); err2 != nil {
+			return nil, fmt.Errorf("failed to remove hostport for container %s in sandbox %s: %v",
```

Comment:
I agree that warning would be better. I copied from TearDownPod below and didn't notice when it changed on rebase. (Though I remember resolving the merge conflict. Oops.) Updating now.